fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1382,6 +1388,7 @@ private void transferToMaster(FrontendNodeType oldType) {
             startMasterOnlyDaemonThreads();
             // start other daemon threads that should running on all FE
             startNonMasterDaemonThreads();
+            insertOverwriteJobManager.cancelRunningJobs();
```

Comment:
why need to cancel running jobs when FE `transferToMaster`?

Reply:
because I think the unfinished insert overwrite job should fail when master failover.

Reply:
Why after master failover, the insert overwrite job should fail.  I think which load job, after master failover, we should continue run the job.

Reply:
I test insert into, when master failover, it will fail. so I keep the same with it. And more, if we continue the job, we should make insert overwrite reenterable, which will need more state and is more complicated.